roles/vagrant/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+- name: Create the hosts file for all machines
+  template:
+    src: etc/hosts.j2
+    dest: /etc/hosts
+    
```

Comment:
It seems to me that setting up hosts files would be generally useful; even outside of using Vagrant.  This role should be split off into a new PR, maybe `hosts-file`?